packages/components/src/fonts.ts

```diff
@@ -9,91 +9,89 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
-import { css } from 'styled-components'
-
 export const family = "'Noto Sans', sans-serif"
 
 /* stylelint-disable opencrvs/no-font-styles */
-const hero = css`
+const hero = `
   font-family: ${family};
   font-weight: 600;
   font-size: 56px;
   line-height: 120%;
```

Comment:
These ones had issues with the `css`, as it interpreted the ${family} as being some kind of CSS value and added commas around it. Thought it's safer to just treat these as strings for now.